packages/embark-vyper/src/utils/async_extend.js

```diff
@@ -0,0 +1,15 @@
+let async = require('async');
```

Comment:
There is code duplication here, maybe that could go in a package?

`async-wrapper`